
---
string[] object_names
geometry_msgs/Point[] object_positions